Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -230,20 +230,21 @@ class SerializerTest : public ::testing::Test {
   /**
    * Creates entries in the proto that we don't care about.
    *
-   * We ignore create time in our serializer. We never set it, and never read it
-   * (other than to throw it away). But the server could (and probably does) set
-   * it, so we need to be able to discard it properly. The ExpectRoundTrip deals
-   * with this asymmetry.
+   * We ignore transaction and create time in our serializer. We never set them,
```

Comment:
Nit: listing the exact fields duplicates the code and might go stale. Perhaps phrase it in a more generic way, such as _"We ignore certain fields in our serializer"_?

Reply:
Done.